server.js

```diff
@@ -15,12 +15,18 @@ const openai = new OpenAI({
   apiKey: process.env.openaiAPI,
 });
 app.get("/openai", async (req, res) => {
+  try {
   const recipeCountryOfOrigin = req.query.recipe_country_of_origin;
   const isLactoseIntolerant = req.query.is_lactose_intolerant;
   const isVegan = req.query.is_vegan; 
 
   console.log({ isLactoseIntolerant }, { recipeCountryOfOrigin }, { isVegan });
 
+  const prompt = `Provide a recipe for a dish from ${recipeCountryOfOrigin}, taking into account the fact that the user is ${
```

Comment:
Is it just me or the` prompt` variable isn't being used in the `completion `object? I can see it being used in the  `imageResponse ` object though